tst/testinstall/reesmat.tst

```diff
@@ -270,7 +270,7 @@ false
 
 # IsReesMatrixSemigroup: for a Rees matrix subsemigroup with generators, 1
 gap> R := ReesMatrixSemigroup(SymmetricGroup(2), [[(1,2)]]);
-<Rees matrix semigroup 1x1 over Sym( [ 1 .. 2 ] )>
+<Rees matrix semigroup 1x1 over Sym( [ 1, 2 ] )>
```

Comment:
There are quite a few of these kinds of changes in Semigroups and Digraphs, (almost?) all of which involve `Sym( [ 1 .. 2 ] )` becoming `Sym( [ 1, 2 ] )`